src/ray/protobuf/gcs_service.proto

```diff
@@ -248,6 +248,13 @@ message AddTaskLeaseRequest {
 message AddTaskLeaseReply {
 }
 
+message AttemptTaskReconstructionRequest {
```

Comment:
Why add the the prefix `Attempt`?

Reply:
This function will try to reconstruct task.